### DISCUSSION TITLE
Response Header

### DISCUSSION THREAD

**Discussion by trex3648** [thunderclient/thunder-client-support#1356] [Q&A]
I want to set a Response Header to a env variable in Test. I can't find any documentation to do this. I do something similar in Postman Tests with the below:

var locationHeader = postman.getResponseHeader("Location");
var id=locationHeader.substring((locationHeader.lastIndexOf("/") + 1),locationHeader.length);
postman.setEnvironmentVariable("LockId",id);
postman.setEnvironmentVariable("lockid",id);
postman.setGlobalVariable("LockId",id);

Is there a way to do something similar in Thunder Client?

**Comment by rangav** [comment]
Hi @trex3648 

can you please answer below

- Are You using free version or paid version 
- What is your team size using TC
- Have you read free version terms - https://www.thunderclient.com/terms#free-plan
